tests/dummy/config/targets.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const browsers = [
+  'last 1 Chrome versions',
+  'last 1 Firefox versions',
+  'last 1 Safari versions'
+];
+
+const isCI = !!process.env.CI;
+const isProduction = process.env.EMBER_ENV === 'production';
+
+if (isCI || isProduction) {
+  browsers.push('ie 11');
```

Comment:
I realize it's only used for testing. But maybe it's better to stay on ie9 in order to fit ember@2 browser support matrix?

Reply:
Hm, if you really want to make sure it works with IE9 (only for Ember 2.x obviously), you would have to run test with saucelabs/browserstack, as this setting won't help you with that. 

Tbh, in a following PR I already prepared to enable the new testing APIs here, I removed IE completely from the dummy app's targets, to enable tests to run with native async/await in headless Chrome instead of the nasty regenerator stuff...